low-level/files/opt/seagate/sspl/bin/sspl_provisioner_init

```diff
@@ -82,8 +82,8 @@ setup_provisioner_prereq(){
     fi
 
     # to install updated packages
-    pkg_name="cortx-prvsnr-cli-1.0.0"
-    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/dev/rhel-7.7.1908/provisioner_last_successful"
+    pkg_name="cortx-prvsnr-cli-1.0.1"
+    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/cortx-1.0/centos-7.8.2003/last_successful"
```

Comment:
in cortx-1.0 branch we need these changes ? its any sspl dev script, but still why we need, provisioner changed in cortx-1.0?

Reply:
To get proper "cortx-prvsnr-cli" I have made these changes. This script only dev purpose. (It is nothing but sspl_init.)

Reply:
sure, not bothered much becoz of dev script only